build_all.sh

```diff
@@ -29,8 +29,8 @@ cp configs/wifi_esp8266_v4.json ./mbed_app.json
 cp configs/eth-wifi-mbedignore ./.mbedignore
 mbed compile -m K64F -t $TOOL
 cp BUILD/K64F/$TOOL/mbed-os-example-client.bin k64f-$TOOL-esp-wifi-v4.bin
-mbed compile -m NUCLEO_F439ZI -t $TOOL
-cp ./BUILD/NUCLEO_F439ZI/$TOOL/mbed-os-example-client.bin f439zi-$TOOL-esp-wifi-v4.bin
+mbed compile -m NUCLEO_F429ZI -t $TOOL
```

Comment:
Why replace the f439zi with 429zi? It is another board, so it should be added here instead.